test/cli/iGenVar_cli_test.cpp

```diff
@@ -209,8 +210,8 @@ TEST_F(iGenVar_cli_test, test_verbose_option)
         "Start clustering...\n"
     };
     EXPECT_EQ(result.exit_code, 0);
-    EXPECT_EQ(result.out.erase(32,19), expected_res_default);
     EXPECT_EQ(result.err, expected_err + expected_err_default_no_err_2);
+    EXPECT_EQ(result.out.erase(408,19), expected_res_default); // erase the filedate
```

Comment:
The standard always uses a space after comma:
```suggestion
    EXPECT_EQ(result.out.erase(408, 19), expected_res_default); // erase the file date
```